.github/actions/build-push-image/action.yaml

```diff
@@ -44,6 +48,11 @@ inputs:
     default: "false"
 runs:
   using: "composite"
+  env:
+    TAG1: "${{ inputs.repository }}:${{ inputs.version }}"
+    TAG2: "${{ inputs.push_to_quay && format('quay.io/{0}:{1}', inputs.repository, inputs.version) || '' }}"
+    TAG3: "${{ inputs.certified_version != '' && format('{0}:{1}', inputs.repository, inputs.certified_version) || '' }}"
+    TAG4: "${{ inputs.certified_version != '' && inputs.push_to_quay && format('quay.io/{0}:{1}', inputs.repository, inputs.certified_version) || '' }}"
```

Comment:
I think it missing the "-certified" ending for the certified image tag. E.g. "1.9.0-certified". So that it won't be overwritten by daily builds

Reply:
Are you sure?

This input is filled like this in the invoking workflow:
`certified_version: ${{ steps.tag.outputs.certified_version }}`

Anyway this code is going to disappear in the next update from here, but will be used similarly later. I will double check we have the expected '-certified' suffix in place.

Reply:
The release process to quay.io for the certified operator bundle should use `<version>-certified` tag format. See https://quay.io/repository/mongodb/mongodb-atlas-kubernetes-operator?tab=tags Otherwise, the daily build would rewrite the <version> and it can not be pulled later by the OLM, since for certification we use SHA256 of an image instead of a tag

Reply:
I am confused:

`.github/workflows/release-post-merge.yml`
Line 97: `certified_version: ${{ steps.tag.outputs.certified_version }}`
This is set at line 67:
`certified_version="${version}-certified"`

So the "<version>-certified" is propagated to the build-and-push sub action and used there.

What am I missing?

Reply:
Test sample:
https://github.com/mongodb/mongodb-atlas-kubernetes/actions/runs/6572673633/job/17854215674

```shell
...
PUSH: docker push quay.io/mongodb/mongodb-atlas-kubernetes-operator-prerelease:1.9.1-rc0
...
PUSH: docker push mongodb/mongodb-atlas-kubernetes-operator-prerelease:1.9.1-rc0-certified
...
PUSH: docker push quay.io/mongodb/mongodb-atlas-kubernetes-operator-prerelease:1.9.1-rc0-certified
...
```